EventsExpress/ClientApp/src/components/tracks/track.js

```diff
@@ -0,0 +1,78 @@
+import React, {Component} from 'react';
+import TrackList from './track-list';
+import TracksFilter from './tracks-filter';
+import Spinner from '../spinner';
+import getAllTracks, {getEntityNames} from '../../actions/tracks/track-list-action';
+import {getFormValues, reset} from 'redux-form';
+import {connect} from 'react-redux';
+
+class Tracks extends Component {
+
+    componentDidMount = () => {
+        this.props.getAllTracks(this.props.tracks.filter);
+        this.props.getEntityNames();
+    }
+
+    handleSubmit = async (filters) => {
```

Comment:
Please merge handle submit and handle page change into single method, since logic is duplicated.